lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -13,13 +13,15 @@
 
 from student.models import CourseEnrollment, UserTestGroup
```

Comment:
I don't think either of these are used here. So this is a reminder to just double check your imports and if there are any that aren't being used, remove them.